server/models/config/index.js

```diff
@@ -0,0 +1,8 @@
+const Airtable = require('airtable');
+require('dotenv').config();
+
+Airtable.configure({
+  endpointUrl: 'https://api.airtable.com',
+  apiKey: process.env.AIRTABLE_API_KEY,
+});
+// const base = Airtable.base(process.env.AIRTABLE_BASE_ID);
```

Comment:
You can shorten these lines of code as follows:
```js
require('dotenv').config();
const base = require('airtable').base(process.env.BASE_ID);
```

Because the default configuration of the Airtable takes the AIRTABLE_API_KEY from environment variable when you requiring it

Reply:
What is better?
```
const airtable = require('airtable');
const base = airtable.base(process.env.AIRTABLE_BASE_ID);
```
Or 
```
const base = require('airtable').base(process.env.BASE_ID);
```
Can it be the same ! Or less code is better ?

Reply:
they are exactly the same in terms of performance and synchronization.
for me, I prefer using the first one as it's more readable